config/locales/vendor_outage/en.yml

```diff
@@ -9,17 +9,18 @@ en:
           address by mail instead.
       pinpoint:
         idv:
-          header: We can only verify your identity by mail right now
-          message_html: Unfortunately, %{app_name} is having technical difficulties. We’re
-            working to resolve an error so that we can verify your identity for
-            <strong>%{sp_name}</strong> quickly.
+          header: We are working to resolve an error
+          message_html: <strong>%{sp_name}</strong> needs to make sure you are you — not
+            someone pretending to be you.
+
+            <p>Unfortunately, we’re having technical difficulties
+            right now. <a href="%{status_page_url}" class="usa-link
+            usa-link--external" target="_blank">Get updates on our status
+            page</a> to learn when the error is resolved.</p>
```

Comment:
1. I'd consider breaking this into multiple separate translation values so we can have less HTML
2. I would pull the link title into its own key and insert that via nested `%{link_html}` interpolation, that way we can build the link with the right class in Ruby, so like

```
<%= t('someone_is_pretending_html', sp_name_html: content_tag(:strong, sp_name)) %>=
<p><%= t('unforunately_html', link_html(new_tab_link_to(t('get_updated'))status_page_url) %></p>
```

In general, I think that putting HTML through translation services can be error-prone, so I try to keep HTML outside of the strings